pkg/db/migrations/202412181141_alter_event_instances.go

```diff
@@ -0,0 +1,45 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+
+	"github.com/go-gormigrate/gormigrate/v2"
+)
+
+func alterEventInstances() *gormigrate.Migration {
+	type EventInstance struct {
+		EventID     string `gorm:"index:idx_status_event_instance"` // primary key of status_events table
+		InstanceID  string `gorm:"index:idx_status_event_instance"` // primary key of server_instances table
+		SpecEventID string `gorm:"index"`                           // primary key of events table
+	}
+
+	return &gormigrate.Migration{
+		ID: "202412181141",
+		Migrate: func(tx *gorm.DB) error {
+			if err := tx.AutoMigrate(&EventInstance{}); err != nil {
+				return err
+			}
+
+			return CreateFK(tx, fkMigration{
+				"event_instances", "server_instances", "instance_id", "server_instances(id)", "ON DELETE CASCADE",
+			}, fkMigration{
+				"event_instances", "status_events", "event_id", "status_events(id)", "ON DELETE CASCADE",
+			})
```

Comment:
would you alse add foreign key for `spec_event_id` on `events` table?

Reply:
no, do you need it?

Reply:
added